scripts/maintenance/git-fame.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
+# git-fame allows printing "loc" (lines of code), "coms" (number of commits) and "fils" (number of files)
+# per contributor (using .mailmap)
+#
+# Suggested usage of git fame:
+#-w, --ignore-whitespace  Ignore whitespace when comparing the parent's
+#                           version and the child's to find where the lines
+#                           came from [default: False].
+#  -M  Detect intra-file line moves and copies [default: False].
+#  -C  Detect inter-file line moves and copies [default: False].
+#
+# These slow it down dramatically, but a lot of this has happened in STIR
+git fame -wMC --format json \
```

Comment:
```suggestion
git fame -wMC --format json --excl '\.(eps|root|v|rtf|gz|scn|s|if|l|pdf|safir|png|.... a lot more but i gave up < 1k lines ....)$' \
```

feel free to keep editing this comment before merging

Reply:
I agree with this suggestion.

Reply:
If you exlude .s and .v then you could exclude .hs and .hv, too. 
They are automatically generated, anyways.

Reply:
I've added most of those. I think best to leave `.par` (example files, important for documentation and not automatic) and `.htm` (release notes).

@casperdcl, can we exclude everything in `external_helpers` (commit of external code, as I can't face subprojects)

Reply:
yes just edited to exclude `external_helpers` too

Reply:
great. I'll commit and give it a go.